.gitignore

```diff
@@ -0,0 +1,25 @@
+# Compiled source #
+###################
+*.class
+
+# Packages #
+############
+*.jar
```

Comment:
Please update your branch, 'cause this file already in develop branch

Reply:
Tried to merge develop into my branch, but it said, that all files are already up to date